src/main/java/com/elixr/poc/service/UserService.java

```diff
@@ -0,0 +1,49 @@
+package com.elixr.poc.service;
+
+import com.elixr.poc.data.User;
+import com.elixr.poc.repository.UserRepository;
+import com.elixr.poc.rest.request.UserRequest;
+import com.elixr.poc.rest.response.ExceptionHandler;
+import com.elixr.poc.rest.response.UserResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+/* service class for  editing response and save or update new user to database */
+@Service
+public class UserService {
+
+    private final UserRepository userRepository;
+
+
+    public UserService(UserRepository saveDataToRepository) {
+
+        this.userRepository = saveDataToRepository;
+    }
+
+    /* Sending the successful response to user */
+    public UserResponse sendResponse(UserRequest userRequestObject) {
```

Comment:
Rename the method to a meaningful name, the purpose of the method seems like create a user

Reply:
changed the method name to createValidUser.